stac_ipyleaflet/stac_discovery/stac.py

```diff
@@ -369,7 +404,10 @@ def get_tile_url(
             kwargs.pop("TileMatrixSetId")
 
         if isinstance(titiler_stac_endpoint, str):
-            r = make_get_request(f"{titiler_stac_endpoint}/{data_type}/{TileMatrixSetId}/tilejson.json", kwargs).json()
+            r = make_get_request(
+                f"{titiler_stac_endpoint}/{data_type}/{TileMatrixSetId}/tilejson.json",
+                kwargs,
+            ).json()
             return r
         else:
-            return "STAC ENDPOINT IS NECESSARY."
\ No newline at end of file
+            return "STAC ENDPOINT IS NECESSARY."
```

Comment:
🔧 : This should be a log statement as we are reporting something to the user. I see we use this return output to print and to check for a str [here](https://github.com/MAAP-Project/stac_ipyleaflet/blob/main/stac_ipyleaflet/stac_discovery/stac_widget.py#L576-L577). Usually the convention is to `print` when debugging and to `log` when showing some type of error or warning to the user.

I did see this existed here already though so if you prefer not to make the change in this PR that is fine w/me

Reply:
I think this was highlighted because of the black formatter lol

Reply:
Yes - unfortunately, a lot of code changes came through because we haven't standardized the formatting. Perhaps we can 
- revisit this in a later clean up PR
- format all of our code separately, before the _next_ PR